src/compose/index.js

```diff
@@ -69,6 +74,11 @@ const enableBatchDispatch = reducer => (state, action) => {
   return reducer(state, action)
 }
 
+const devTools = () =>
```

Comment:
I'm tempted to say we should use the HAS_DEBUG_FLAG check here instead so that you could leave this in and use it to debug your production app too.

Reply:
What do you think? If we did, then we could leave the capability in place, but only if someone knew to add the `localStorage.debug` flag on would it actually initiate in production.

Reply:
I really like that idea. There's been a few times I wish I had access to the devtool in prod.

Reply:
I like that too! I added in both options:

Devtools is always enabled (if you have the extension) in development mode
Devtools is enabled if debug mode is enabled, regardless of env

Thoughts?

Reply:
I think that's good there's really no reason to turn it off in dev, I just had a small style suggestion, but I think the idea is solid.